glibc/riscv64.patch

```diff
@@ -1,31 +1,47 @@
---- PKGBUILD
-+++ PKGBUILD
-@@ -7,13 +7,13 @@
+Index: PKGBUILD
+===================================================================
+--- PKGBUILD    (revision 448151)
++++ PKGBUILD    (working copy)
+@@ -7,7 +7,7 @@
  # NOTE: valgrind requires rebuilt with each major glibc version
  
  pkgbase=glibc
 -pkgname=(glibc lib32-glibc)
 +pkgname=(glibc)
  pkgver=2.35
- pkgrel=2
+ _commit=d7d1eebd4d5806be55ffacbf18917ad68d4ae7fd
+ pkgrel=6
+@@ -14,13 +14,11 @@
  arch=(x86_64)
  url='https://www.gnu.org/software/libc'
  license=(GPL LGPL)
 -makedepends=(git gd lib32-gcc-libs python)
 +makedepends=(git gd python)
- optdepends=('perl: for mtrace')
- options=(!strip staticlibs !lto)
- #_commit=3de512be7ea6053255afed6154db9ee31d4e557a
-@@ -36,7 +36,7 @@ b2sums=('623c728884f070cd87ffeb9203f74206197c52405ac9bc44f3dd519a3468b8e8ae2536c
+ options=(debug staticlibs !lto)
+ source=(git+https://sourceware.org/git/glibc.git#commit=${_commit}
+         locale.gen.txt
+         locale-gen
+-        lib32-glibc.conf
+-        sdt.h sdt-config.h
+         disable-clone3.diff
+ )
+ validpgpkeys=(7273542B39962DF7B299931416792B4EA25340F8 # Carlos O'Donell
+@@ -28,13 +26,10 @@
+ b2sums=('SKIP'
+         '46d533d25c7a2ce4ae75d452eee7ebb8e3ce4d191af9be3daa43718b78cb81d33cfd8046a117a15d87de9f5e940448c66005b0490515bf731c9e4691c53908d6'
+         '04fbb3b0b28705f41ccc6c15ed5532faf0105370f22133a2b49867e790df0491f5a1255220ff6ebab91a462f088d0cf299491b3eb8ea53534cb8638a213e46e3'
+-        '7c265e6d36a5c0dff127093580827d15519b6c7205c2e1300e82f0fb5b9dd00b6accb40c56581f18179c4fbbc95bd2bf1b900ace867a83accde0969f7b609f8a'
+-        'a6a5e2f2a627cc0d13d11a82458cfd0aa75ec1c5a3c7647e5d5a3bb1d4c0770887a3909bfda1236803d5bc9801bfd6251e13483e9adf797e4725332cd0d91a0e'
+-        '214e995e84b342fe7b2a7704ce011b7c7fc74c2971f98eeb3b4e677b99c860addc0a7d91b8dc0f0b8be7537782ee331999e02ba48f4ccc1c331b60f27d715678'
          'edef5f724f68ea95c6b0127bd13a10245f548afc381b2d0a6d1d06ee9f87b7dd89c6becd35d5ae722bf838594eb870a747f67f07f46e7d63f8c8d1a43cce4a52')
  
  prepare() {
 -  mkdir -p glibc-build lib32-glibc-build
 +  mkdir -p glibc-build
  
-   [[ -d glibc-$pkgver ]] && ln -s glibc-$pkgver glibc 
+   [[ -d glibc-$pkgver ]] && ln -s glibc-$pkgver glibc
```

Comment:
Please leave them as is. Disable some functions of your text editor temporarily?